content/rooms/fragments/v11-redactions.md

```diff
@@ -0,0 +1,41 @@
+---
+---
+
+{{% added-in this=true %}} The top-level `origin`, `membership`, and `prev_state` properties
+are no longer protected from redaction. The [`m.room.create`](/client-server-api#mroomcreate)
+event now keeps the entire `content` property. The [`m.room.redaction`](/client-server-api#mroomredaction)
+event keeps the `redacts` property under `content`. The
+[`m.room.power_levels`](/client-server-api#mroompower_levels) event keeps the
+`invite` property under `content`.
+
+The full redaction algorithm follows.
```

Comment:
These two paragraphs won't be appropriate for inclusion in room v12 (since they document the changes since room v10). Can you move them into the main `rooms/v11.md` file?

(Looks like room v9/v10 messed this up. Room v6/v7 did better: there is a paragraph in https://spec.matrix.org/v1.7/rooms/v6/#redactions which is not present in https://spec.matrix.org/v1.7/rooms/v7/#redactions)

Reply:
I also fixed room v9/v10.